styleguide/js/zoolander.js

```diff
@@ -111,8 +111,42 @@ jQuery(document).ready(function($) {
     });
 
     $('#footer-accordion').tabCollapse();
+    //initialize the responsive tables
+    $('.compareTable').responsiveTable();
 });
 
+//responsive table function
+(function($){
+  'use strict';
+  $.fn.responsiveTable = function(){
+    var $table = $(this);
+    var $window = $(window);
+    $window.resize(initTable);
+    initTable();
+    function initTable(){
+      var cellTitle = [];
+      var $viewPort = parseInt($window.outerWidth());
+      $table.each(function(){
+        var $el = $(this);
+        if ($viewPort < 768){
+          $el.find('.compareTable-theadCell').each(function(){
+            cellTitle.push($(this).text());
+          });
+          $el.find('.compareTable-bodyCell').each(function(){
+            $(this).find('b.compareTable-bodyCell-mobileTitle').remove();
+            $(this).prepend('<b class="compareTable-bodyCell-mobileTitle">' + cellTitle[$(this).index()] + ': </b>');
+          });
+        }
+        else if ($viewPort > 768) {
+          $el.find('.compareTable-bodyCell b.compareTable-bodyCell-mobileTitle').each(function(){
+            $(this).remove();
+          });
+        }
+      });
+    }
+    return this;
+  };
+}(jQuery));
```

Comment:
Can you provide some in-code comments here explaining what exactly you're doing as that will be easier to parse what's actually going on here.